back-end/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:16
```

Comment:
This has to be slightly changed to match the one that’s in development, I refactored the way it copy’s the subdirectories! Either I can make the change or you can just copy the one from development whatever you prefer!

Reply:
I copied the one from development. Thanks!